File: RSA/RSA-m&n/RSA_M&N/RSA_M&N/ViewController.m
//
//  ViewController.m
//  RSA_M&N
//
//  Created by 郑洪益 on 2022/6/12.
//

#import "ViewController.h"
#import "RSA.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    NSString* result = [RSA encryptString:@"123" mod:@"a8b9fe573e72e838e9de31d588acecf04cc15850006e0f090f99cf1b45175e769c8530172689e5beb7604d1baf3ba0dcf833a59df62aaa1fabdf79a6b2e3f429" exp:@"010001"];
    NSLog(@"result:%@", result);
}

@end

File: YIHFWebView/YIHFWebView/ViewController.swift
//
//  ViewController.swift
//  YIHFWebView
//
//  Created by zhenghongyi on 2020/10/15.
//

import UIKit
import WebKit
import SnapKit

class ViewController: UIViewController {

    let hfWebView = YIHFWebView()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        view.addSubview(hfWebView)
        hfWebView.snp.makeConstraints { (make) in
            make.edges.equalTo(self.view)
        }
        
        let headerView = UIView()
        headerView.backgroundColor = .yellow
        let footerView = UIView()
        footerView.backgroundColor = .green
        let webView = WKWebView()

        if let path = Bundle.main.path(forResource: "1", ofType: "html"),
           let html = try? String(contentsOfFile: path) {
            webView.loadHTMLString(html, baseURL: nil)
        }

        hfWebView.config(head: headerView, web: webView, foot: footerView)

        // 用于获取数据后的展示，或者高度展开收缩
        hfWebView.footHeight = 100
        hfWebView.headHeight = 100
    }

}

